apps/backend/src/modules/project-groups/project-group.controller.ts

```diff
@@ -70,7 +70,13 @@ class ProjectGroupController extends BaseController {
 				),
 			method: "POST",
 			path: ProjectGroupsApiPath.ROOT,
-			preHandlers: [checkUserPermissions([PermissionKey.MANAGE_USER_ACCESS])],
+			preHandlers: [
+				checkUserPermissions([
+					PermissionKey.MANAGE_PROJECT,
+					PermissionKey.VIEW_ALL_PROJECTS,
```

Comment:
User with only View All Projects permission shouldn't be able to create project group